functions/send-email/1.0/index.js

```diff
@@ -0,0 +1,56 @@
+import templayed from './templayed';
+
+const sendEmail = async ({
+  host,
+  port,
+  username,
+  password,
+  secure,
+  senderEmail,
+  senderName,
+  toEmail,
+  toName,
+  subject,
+  body,
+  record,
+  variables,
+}) => {
+  const smtpCredentials = {
+    host,
+    port,
+    username,
+    password,
+    secure,
+  };
+
+  const parsedBody = body;
```

Comment:
Please remove this variable, since you can use body as a parameter for templayed()